docs/guide/topics/embeds.rst

```diff
@@ -0,0 +1,318 @@
+    )
+
+.. tip::
+
+    There is also a ``color`` parameter, and respective aliases for ``discord.Color``.
+    In case you prefer that spelling.
+
+Next, we'll set the title of the embed.
+
+Just like ``colour``, it's another parameter of the constructor, called ``title``.
+
+.. code-block:: python3
+    :emphasize-lines: 3
+
+    my_embed = discord.Embed(
+        colour=discord.Colour.purple(),
+        title="Hello, World!",
+    )
+
+Let's see how that looks so far if we send it.
+
+.. code-block:: python3
+
+    await channel.send(embed=my_embed)
+
+
+.. image:: /images/guide/topics/embeds/simple_embed_2.png
+    :scale: 38%
+
+So close! Now, all we need to do is add the description.
+
+.. code-block:: python3
+
+    my_embed = discord.Embed(
+        colour=discord.Colour.purple(),
+        title="Hello, World!",
+        description="This bot is running on discord.py!"
+    )
+
+.. image:: /images/guide/topics/embeds/simple_embed_final.png
+    :scale: 38%
+
+And that's it!
+
+As you can see, the interface that discord.py provides is essentially a direct mapping of what you see in the Discord client.
+
+To summarise, we call the constructor for :class:`Embed` and provide the following keyword arguments:
+
+- ``colour`` to set the colour of the embed.
+- ``title`` to set the title of the embed.
+- ``description`` to set the description of the embed.
+
+Let's take a look at what else we can add to an embed.
+
+Using Embed Components
+-----------------------
+
+The :class:`Embed` class allows usage of the `factory` pattern.
+
+.. hint::
+
+    This means that certain methods will return a modified instance of the embed class,
+    so you can chain method calls.
+
+    We will discover this as we go through the guide.
+
+Markdown
+~~~~~~~~~
+
+An embed's description and fields allow you to use markdown, so
+that includes **this**, *that*, and even ``this`` -- read more about it at :ref:`_guide_topic_markdown`.
+
+Usually, in a message's content it is not possible to send hyperlinks.
+However, embeds allow you to include them! So you can do little tricks like:
+
+.. code-block:: python3
+
+    my_embed.description = """**Hello**! My name is Danny!
+    Check out my GitHub: [click here!](https://github.com/Rapptz)
+    I have a page dedicated to [`C++ Tutorials`](https://rapptz.github.io/cpptuts)!
+    """
+
+.. image:: /images/guide/topics/embeds/hyperlink_description_embed.png
+    :scale: 38%
+
+.. tip::
+
+    Hyperlinks in markdown follow a specific format: ``[text](link)``.
+
+It is also possible to add tooltips to hyperlinks, which are displayed when you hover over the hyperlink.
+
+.. code-block:: python3
+
+    my_embed.description = """**Hello**! My name is Danny!
+    Check out my GitHub: [click here!](https://github.com/Rapptz "My GitHub")
+    I have a page dedicated to [`C++ Tutorials`](https://rapptz.github.io/cpptuts)!
+    """
+
+.. image:: /images/guide/topics/embeds/hyperlink_tooltip_description_embed.png
```

Comment:
Tiny tooltip! I know this is sort of inevitable since we're doing high quality screenshots, but it might even be worth photoshopping the tooltip to be larger so it looks a little less weird.